README.md

```diff
@@ -40,126 +40,163 @@ worker:
+ `invenio.init`                                      | Whether to initiate database, index and roles                                                       | `false`               
+ `invenio.default_users`                             | If set, create users identified by email:password on install (only works if init=true)              | `nil`                 
+ `invenio.demo_data`                                 | Whether to create demo data on install (only works if init=true and if `default_users` isn't empty) | `false`               
+ `invenio.sentry.enabled`                            | Enable Sentry logging                                                                               | `false`               
+ `invenio.sentry.existing_secret`                    | Whether to use an existing secret or create a new one                                               | `false`               
+ `invenio.sentry.secret_name`                        | Name of the secret to use or create                                                                 | `sentry-secrets`      
+ `invenio.sentry.dsn`                                | DSN for sentry                                                                                      | `""`                  
+ `invenio.datacite.enable`                           | Enable DataCite provider                                                                            | `false`               
+ `invenio.datacite.existing_secret`                  | Whether to use an existing secret or create a new one                                               | `false`               
+ `invenio.datacite.secret_name`                      | Name of the secret to use or create                                                                 | `datacite-secrets`    
+ `invenio.remote_apps.enabled`                       | Enable logging with remote applications                                                             | `false`               
+ `invenio.remote_apps.existing_secret`               | Whether to use an existing secret or create a new one                                               | `false`               
+ `invenio.remote_apps.secret_name`                   | Name of the secret to use or create                                                                 | `remote-apps-secrets` 
+ `invenio.remote_apps.credentials`                   | List of remote applications' credentials (name, consume_key, consumer_secret)                       | `""`                  
+ `invenio.extra_config`                              | List of key:value configuration variables accepted by the python application                        | `""`                  
+ `invenio.file_config_maps`                          | List of objects describing additional config maps read from file                                    | `""`                  
```

Comment:
May I suggest improving the name `file_config_maps` and the related text? What do you mean with `read from file`? In which case would I use that?
It is hard to understand the difference with the above `extra_config`.

I am wondering: is it really `invenio` config? Or could it be any ConfigMap containing files? Maybe a quick chat IRL will be faster here.